seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -183,27 +190,35 @@ def test_create_project_from_workspace(self, mock_file_iter, mock_file_exist, mo
         mock_has_service_account.assert_not_called()
-        mock_sleep.assert_not_called()
-        mock_file_exist.assert_called_with('gs://test_bucket/test_path.vcf', user=self.manager_user)
-        mock_file_iter.assert_called_with('gs://test_bucket/test_path.vcf', byte_range=None)
-
-        email_body = """
-        test_user_manager@test.com requested to load WES data (GRCh38) from AnVIL workspace "{namespace}/{name}" at 
-        "gs://test_bucket/test_path.vcf" to seqr project {{project_name}} (guid: {guid})
-
-        The sample IDs to load are attached.    
-        """.format(namespace=TEST_WORKSPACE_NAMESPACE, name=TEST_NO_PROJECT_WORKSPACE_NAME, guid=project.guid)
-        mock_email.assert_called_with(
-            subject='AnVIL data loading request',
-            body=email_body.format(project_name=TEST_NO_PROJECT_WORKSPACE_NAME),
-            to=['test_data_manager@test.com', 'test_superuser@test.com'],
-            attachments=[('{}_sample_ids.tsv'.format(project.guid), 'NA19675\nNA19678\nHG00735')]
+        mock_time.sleep.assert_not_called()
+        mock_file_exist.assert_called_with(TEST_VCF_PATH, user=self.manager_user)
+        mock_file_iter.assert_called_with(TEST_VCF_PATH, byte_range=None)
+        mock_tempfile.NamedTemporaryFile.assert_called_with(mode='wb', delete=False)
+        mock_tempfile.NamedTemporaryFile.return_value.__enter__.return_value.write.assert_called_with(b's\nNA19675\nNA19678\nHG00735')
+        mock_mv_file.assert_called_with(
+            TEMP_PATH, 'gs://seqr-datasets/v02/GRCh38/AnVIL_WES/{guid}/base/{guid}_ids.txt'.format(guid=project.guid),
+            user=self.manager_user
         )
-        html_project_name = '<a href="http://testserver/project/{guid}/project_page">{name}</a>'.format(
-                name=TEST_NO_PROJECT_WORKSPACE_NAME, guid=project.guid)
-        mock_email.return_value.attach_alternative.assert_called_with(
-            email_body.format(project_name=html_project_name), 'text/html')
-        mock_email.return_value.send.assert_called()
+        slack_message = """
+        *{user}* requested to load WES data (GRCh38) from AnVIL workspace *{namespace}/{name}* at 
+        {vcf_path} to seqr project <http://testserver/project/{guid}/project_page|*anvil-no-project-workspace1*> (guid: {guid})  
+  
+        The sample IDs to load have been uploaded to gs://seqr-datasets/v02/GRCh38/AnVIL_WES/{guid}/base/{guid}_ids.txt.  
+  
+        DAG for the loading pipeline:
+        ```{{
+    "active_projects": [
+        "{guid}"
+    ],
+    "vcf_path": "{vcf_path}",
+    "project_path": "gs://seqr-datasets/v02/GRCh38/AnVIL_WES/{guid}/v1",
+    "projects_to_run": [
+        "{guid}"
+    ]
+}}```
+        """.format(user=self.manager_user.email, namespace=TEST_WORKSPACE_NAMESPACE, name=TEST_NO_PROJECT_WORKSPACE_NAME,
+                   guid=project.guid, vcf_path=TEST_VCF_PATH)
```

Comment:
in tests wherever possible its better to hardcode the strings rather than string format. Other than guid I think you should put the literal strings in the template instead of formatting them here